remoting/getty/config.go

```diff
@@ -188,6 +196,12 @@ func (c *ClientConfig) CheckValidity() error {
 			c.HeartbeatPeriod, time.Duration(config.MaxWheelTimeSpan))
 	}
 
+	if len(c.HeartbeatTimeout) == 0 {
+		c.heartbeatTimeout = 5 * time.Second
```

Comment:
The default hb timeout duration 5s is too short. Pls change it to 90s.

Reply:
that is a test data， I will change it to 60s same as `Dubbo`